cdc_vaccines/delphi_cdc_vaccines/run.py

```diff
@@ -69,8 +68,8 @@ def run_module(params):
         df["val"] = df[["geo_id", sensor]].groupby("geo_id")[sensor].transform(
             smoother[0].smooth
         )
-        df["se"] = np.nan
-        df["sample_size"] = np.nan
+        df["se"] = nancodes.Nans.NOT_APPLICABLE
+        df["sample_size"] = nancodes.Nans.NOT_APPLICABLE
```

Comment:
This is more what I meant
```suggestion
        df["se"] = np.nan
        df["sample_size"] = np.nan
        df["missing_se"] = nancodes.Nans.NOT_APPLICABLE
        df["missing_sample_size"] = nancodes.Nans.NOT_APPLICABLE
```

Reply:
Sorry! I messed up version control. Thanks for the catch - it should all be taken care of now :)

Reply:
It looks like create_export_csv in delphi_utils needs the 'se' and 'sample_size' columns - not the missing columns - should I change the delphi utils function? 

../../_delphi_utils_python/delphi_utils/export.py:73: in create_export_csv
    export_df = df[df["timestamp"] == date][["geo_id", "val", "se", "sample_size",]]

Reply:
Okay, I'll wait until that gets merged in.

Reply:
Now that #1252 has been merged, my suggestion should work